src/apps/Agreement/AgreementDoc.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
What do you think about renaming this to `AgreementDocument`? We generally prefer being explicit :)